ArrtSource/Storage/UI/StorageBrowserModel.cpp

```diff
@@ -263,7 +265,14 @@ bool StorageBrowserModel::IsArrAsset(const QString& file)
 
 bool StorageBrowserModel::IsSrcAsset(const QString& file)
 {
-    return file.endsWith(".gltf", Qt::CaseInsensitive) || file.endsWith(".glb", Qt::CaseInsensitive) || file.endsWith(".fbx", Qt::CaseInsensitive);
+    QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
+    QString formats = ";gltf;glb;fbx;" + env.value("ARRT_ALLOWED_ASSET_FORMATS", "") + ";";
```

Comment:
Just wondering, is this list supposed to show up in the file browser dialog as well?

Reply:
We have a custom file browse dialog for Azure Storage, it just displays all files which match this list. The actual file extensions don't appear in the UI.